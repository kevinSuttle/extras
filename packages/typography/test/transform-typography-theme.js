import lincoln from 'typography-theme-lincoln'
import transform from '../src/transform-typography-theme'

test('returns an object', () => {
  const config = transform(lincoln)
  expect(typeof config).toBe('object')
  expect(config).toMatchSnapshot()
})
