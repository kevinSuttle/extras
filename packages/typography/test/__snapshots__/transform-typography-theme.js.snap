// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns an object 1`] = `
Object {
  "a": Object {
    "backgroundImage": "linear-gradient(to top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0) 1px, #134896 1px, #134896 2px, rgba(0, 0, 0, 0) 2px)",
    "color": "#134896",
    "textDecoration": "none",
    "textShadow": ".03em 0 #fff,-.03em 0 #fff,0 .03em #fff,0 -.03em #fff,.06em 0 #fff,-.06em 0 #fff,.09em 0 #fff,-.09em 0 #fff,.12em 0 #fff,-.12em 0 #fff,.15em 0 #fff,-.15em 0 #fff",
  },
  "a:hover,a:active": Object {
    "backgroundImage": "none",
    "textShadow": "none",
  },
  "abbr": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "abbr[title]": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
    "textDecoration": "none",
  },
  "acronym": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "address": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "b": Object {
    "fontWeight": 700,
  },
  "blockquote": Object {
    "borderLeft": "0.5925rem solid #134896",
    "color": "hsla(0,0%,0%,0.65)",
    "fontSize": "1.1487rem",
    "fontStyle": "italic",
    "lineHeight": "1.58rem",
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": "0.9875rem",
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "blockquote *:last-child": Object {
    "marginBottom": 0,
  },
  "blockquote > :last-child": Object {
    "marginBottom": 0,
  },
  "blockquote cite": Object {
    "color": "hsla(0,0%,0%,0.73)",
    "fontSize": "1rem",
    "fontStyle": "normal",
    "fontWeight": 400,
    "lineHeight": "1.58rem",
  },
  "blockquote cite:before": Object {
    "content": "\\"â€” \\"",
  },
  "body": Object {
    "MozFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "WebkitFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "boxSizing": "border-box",
    "color": "hsla(0,0%,0%,0.73)",
    "font": "118.75%/1.58 'Lora',serif",
    "fontFamily": "'Lora',serif",
    "fontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "fontKerning": "normal",
    "fontWeight": 400,
    "msFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "overflowY": "scroll",
    "wordWrap": "break-word",
  },
  "code": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.58rem",
  },
  "dd": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dl": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dt": Object {
    "fontWeight": 700,
  },
  "fieldset": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "figure": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "form": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "h1": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "2rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h1,h2,h3,h4,h5,h6": Object {
    "marginBottom": "0.79rem",
    "marginTop": "2.37rem",
  },
  "h2": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "1.51572rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h3": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "1.31951rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h4": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "1rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h5": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "0.87055rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h6": Object {
    "color": "hsla(0,0%,0%,0.9)",
    "fontFamily": "'Varela Round',sans-serif",
    "fontSize": "0.81225rem",
    "fontWeight": "400",
    "lineHeight": 1.1,
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "hgroup": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "hr": Object {
    "background": "hsla(0,0%,0%,0.2)",
    "border": "none",
    "height": "1px",
    "marginBottom": "calc(1.58rem - 1px)",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "iframe": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "img": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "maxWidth": "100%",
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "kbd": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.58rem",
  },
  "li": Object {
    "marginBottom": "calc(1.58rem / 2)",
  },
  "li *:last-child": Object {
    "marginBottom": 0,
  },
  "li > ol": Object {
    "marginBottom": "calc(1.58rem / 2)",
    "marginLeft": "1.58rem",
    "marginTop": "calc(1.58rem / 2)",
  },
  "li > p": Object {
    "marginBottom": "calc(1.58rem / 2)",
  },
  "li > ul": Object {
    "marginBottom": "calc(1.58rem / 2)",
    "marginLeft": "1.58rem",
    "marginTop": "calc(1.58rem / 2)",
  },
  "li>ol,li>ul": Object {
    "marginBottom": 0,
    "marginLeft": "20px",
  },
  "noscript": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.58rem",
    "marginLeft": "1.58rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol li": Object {
    "paddingLeft": 0,
  },
  "p": Object {
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "p *:last-child": Object {
    "marginBottom": 0,
  },
  "pre": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.58rem",
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "samp": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.58rem",
  },
  "strong": Object {
    "fontWeight": 700,
  },
  "table": Object {
    "borderCollapse": "collapse",
    "fontSize": "1rem",
    "lineHeight": "1.58rem",
    "marginBottom": "1.58rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "width": "100%",
  },
  "td,th": Object {
    "MozFontFeatureSettings": "\\"tnum\\"",
    "WebkitFontFeatureSettings": "\\"tnum\\"",
    "borderBottom": "1px solid hsla(0,0%,0%,0.12)",
    "fontFeatureSettings": "\\"tnum\\"",
    "msFontFeatureSettings": "\\"tnum\\"",
    "paddingBottom": "calc(0.79rem - 1px)",
    "paddingLeft": "1.05333rem",
    "paddingRight": "1.05333rem",
    "paddingTop": "0.79rem",
    "textAlign": "left",
  },
  "th": Object {
    "fontWeight": 700,
  },
  "th:first-child,td:first-child": Object {
    "paddingLeft": 0,
  },
  "th:last-child,td:last-child": Object {
    "paddingRight": 0,
  },
  "thead": Object {
    "textAlign": "left",
  },
  "ul": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.58rem",
    "marginLeft": "1.58rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ul li": Object {
    "paddingLeft": 0,
  },
}
`;
